tcms/bugs/tests/test_views.py

```diff
@@ -139,3 +140,61 @@ def test_new_bug_assignee_inferred_from_components(self):
         bug_created = Bug.objects.last()
         self.assertEqual(bug_created.summary, self.summary)
         self.assertEqual(bug_created.assignee, comp.initial_owner)
+
+
+class TestEditBug(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'bugs.change_bug')
+
+        cls.bug = BugFactory()
+
+        cls.url = reverse('bugs-edit', args=(cls.bug.pk,))
+
+        cls.summary_edit = 'An edited summary'
+        cls.reporter_edit = cls.tester
+        cls.assignee_edit = UserFactory()
+        cls.product_edit = ProductFactory()
+        cls.version_edit = VersionFactory()
+        cls.build_edit = BuildFactory()
+
+        cls.edit_data = {
+            'summary': cls.summary_edit,
+            'reporter': cls.reporter_edit.pk,
+            'assignee': cls.assignee_edit.pk,
+            'product': cls.product_edit.pk,
+            'version': cls.version_edit.pk,
+            'build': cls.build_edit.pk
+        }
+
+    def test_get_view(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['page_title'], _('Edit bug'))
+        self.assertEqual(response.context['form_post_url'], self.url)
+        self.assertTemplateUsed(response, 'bugs/mutable.html')
```

Comment:
Same comments as above. As-is this scenario is duplicating the permissions scenario. 

Important items for this view are the list of fields to be editted.